.github/workflows/images.yml

```diff
@@ -0,0 +1,79 @@
+name: Publish Container Images
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - v[0-9]+.[0-9]+.[0-9]+
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+permissions:
+  contents: read
+  packages: write
+
+jobs:
+  build-and-push:
+    name: Build and Push (${{ matrix.image }})
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        image: [
+          #default-validator,
+          device-injector,
+          differ,
+          hook-injector,
+          logger,
+          #network-device-injector,
+          #network-logger,
+          template,
+          ulimit-adjuster,
+          v010-adapter,
+          wasm,
```

Comment:
nit: I don't think we should build an image for the wasm sample plugin as well. The only way to activate is to drop or link it in to the stock pre-installed plugin directory then let the NRI core/the runtime discover it, realize it is a WASM binary and load it as a WASM plugin.

Reply:
Check, will remove in an update. I just brainlessly "enabled" all